test/suites/smoke-test-dancelight/test-external-validators-rewards.ts

```diff
@@ -82,13 +83,16 @@ describeSuite({
 
                 // The event is triggered, nonce should be incremented
                 if (event) {
-                    expect(nonceDiff).toEqual(1);
-                    expect(
-                        sovereignBalanceCheckpointA.toBigInt() - sovereignBalanceCheckpointB.toBigInt()
-                    ).to.be.lessThan(0);
+                    const supplyBefore = (await apiAtCheckpointA.query.balances.totalIssuance()).toBigInt();
+                    const sovereignIssuance = (supplyBefore * DANCELIGHT_ERA_INFLATION_PERBILL) / 1_000_000_000n;
 
-                    // The event is not triggered, nonce should be the same
-                } else {
+                    expect(nonceDiff).toEqual(1);
+                    expect(sovereignBalanceCheckpointB.toBigInt() - sovereignBalanceCheckpointA.toBigInt()).to.be.equal(
+                        sovereignIssuance
+                    );
```

Comment:
I would add an error margin here as we have in block inflation, unless we are sure this value will be exact